cellmincer/denoise/argparse.py

```diff
@@ -51,30 +52,30 @@ def add_subparser_args(subparsers: argparse) -> argparse:
         dest='model_type',
         default=None,
         required=True,
-        help='Name of model class.')
+        help='Name of model class. Options: "spatial-unet-2d-temporal-denoiser"')
     
     subparser.add_argument(
-        '--avi_frames',
+        '--avi-frames',
         nargs=2,
         type=int,
         dest='avi_frames',
         default=None,
         required=False,
-        help='Range of frames to output as .AVI.')
+        help='Range of frames to output as .AVI, declared as start (inclusive) and end (exclusive).')
     
     subparser.add_argument(
-        '--avi_sigma',
+        '--avi-sigma',
         nargs=2,
         type=int,
         dest='avi_sigma',
         default=None,
         required=False,
-        help='Pixel intensity clip range of .AVI, as stds of intensity distribution.')
+        help='Pixel intensity clip range of .AVI, as z-scores of intensity distribution.')
```

Comment:
I suggest updating this to two values, `avi-min-z-score` and `avi-max-z-score`. Then, clip the values to `[mu - avi_min_z_score * sigma, mu - avi_max_z_score * sigma]`. This give you more control to set the range for better visualization dynamic range.

Reply:
(these are basically sigma-lo and sigma-hi).